packages/main/src/plugin/configuration-registry.spec.ts

```diff
@@ -49,6 +49,16 @@ vi.mock('node:fs', () => ({
   },
 }));
 
+// mock DefaultConfiguration for the new managed defaults functionality
+vi.mock('./default-configuration.js', () => ({
```

Comment:
nitpick: use the import synthax, this ensure that the import exists, and allow typecheck to validate it


```suggestion
vi.mock(import('./default-configuration.js'), () => ({
```